features/org.aposin.mergeprocessor.thirdparty.feature/feature.xml

```diff
@@ -72,4 +72,19 @@
          version="0.0.0"
          unpack="false"/>
 
+   <!-- After Eclipse upgrade 4.13 jface.text is missing in the product. This may be fixed in the future. --> 
```

Comment:
Feature `org.eclipse.rcp` does not automatically deliver `org.eclipse.text` and `org.eclipse.jface.text`, but are mandatory for execution. Adding additional features to the product results in troubles and more plugin's would unnecessarily be loaded. 

So I decided to add them to org.aposin.mergeprocessor.thirdparty.feature, with the note that this is only a workaround.

Reply:
I will investigate and have a look which feature exactly removed these plugins. Then I raise an issue.

Reply:
My comment was more about opening a new issue in our repository to track that we have this workaround here and then take action whenever we find a solution, without the need to explore the code to check that there is a workaround. Then we can track or add extra information (e.g., bug-report from Eclipse, from which feature came before, etc.) on that ticket.

Reply:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=552134

Reply:
I open also #28 to track the issue.